api/Navigator.json

```
@@ -610,7 +610,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": false,
+              "version_added": "17",
               "notes": "Use <code>window.doNotTrack</code> instead."
```

The note needs to be updated (or we could even introduce a range here that talks about the prior support on window).

```suggestion
              "notes": "Edge prior to version 17 implemented <code>window.doNotTrack</code>."
```

---

Err, sorry about the screwed up indentation in the suggestion here.